util/app-config/src/configs/network.rs

```diff
@@ -90,6 +90,17 @@ impl Config {
             Ok(file) => file,
             Err(_) => return Ok(None),
         };
+        #[cfg(unix)]
+        {
+            use std::os::unix::fs::PermissionsExt;
+            if file.metadata()?.permissions().mode() != 0o600 {
```

Comment:
```suggestion
            if file.metadata()?.permissions().mode() & 0o177 != 0 {
```

0400 should also be accepted.